marti_sensor_msgs/CMakeLists.txt

```
@@ -10,6 +10,7 @@ add_message_files(FILES
   DioPortState.msg
   DioRealTimeData.msg
   Direction.msg
+  DifferentialMeasurement.msg
```

```suggestion
  DifferentialMeasurement.msg
  DioPortState.msg
  DioRealTimeData.msg
  Direction.msg
```

`f` comes before `o`.